user-service/routers/router.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+from fastapi import APIRouter, HTTPException, status, Path, Query, Body, Depends
+from fastapi.responses import JSONResponse, PlainTextResponse
+
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from jose import JWTError, jwt
+
+from pydantic import BaseModel, Field
+
+from services.service import UserService
+
+from config.envs import *
+
+SECRET_KEY = os.getenv(SECRET_KEY_NAME)
+ENCODER_ALGORITHM = os.getenv(ENCODER_ALGORIGHTM_NAME)
+
+ADMIN_USERNAME = os.getenv(USER_DB_VAR_USER_NAME)
+ADMIN_PASSOWRD = os.getenv(USER_DB_VAR_PASSWORD_NAME)
+
+router = APIRouter()
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="api/v1/token")
+
+def is_admin(username, password):
+    if username == ADMIN_USERNAME and password == ADMIN_PASSOWRD:
+        return True
+    return False
+
+def authenticate_user(username, password):
+    if is_admin:
+        return {"username": username, "password": password}
+    
+def create_access_token(data: dict):
+    encoded_jwt = jwt.encode(data.copy(), SECRET_KEY, algorithm=ENCODER_ALGORITHM)
+    return encoded_jwt
+
+def get_current_user(token: str = Depends(oauth2_scheme)):
+    credentials_exception = HTTPException(
+        status_code=status.HTTP_401_UNAUTHORIZED,
+        detail="Could not validate credentials",
+        headers={"WWW-Authenticate": "Bearer"},
+    ) 
+    try:
+        payload = jwt.decode(token, SECRET_KEY, algorithms=[ENCODER_ALGORITHM])
+        if payload.get("username") is None:
+            raise credentials_exception
+    except JWTError:
+        raise credentials_exception
+    return payload.copy()
+
+@router.post("/token")
+def login(form_data: OAuth2PasswordRequestForm = Depends()):
+    user = authenticate_user(form_data.username, form_data.password)
+    if not user:
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="Incorrect username or password",
+            headers={"WWW-Authenticate": "Bearer"},
+        ) 
+    access_token = create_access_token(data=user)
```

Comment:
Currently, this is unhandled. It'll just say internal service error if it cannot create a token. Ideally, we would want it to return something more.